### PR TITLE
fixed compilation error on OpenBSD, missing sockpeercred 

### PR DESCRIPTION
**Pull request by opunix**
next try 👍 

BR Frank

### CODE REVIEW

File: src/platform/posix/posix_ipcconn.c [hunk 1]
```diff
@@ -350,6 +350,17 @@ ipc_conn_peerid(nni_ipc_conn *c, uint64_t *euid, uint64_t *egid,
 	*znid = ucred_getzoneid(ucp);
 	ucred_free(ucp);
 	return (0);
+#elif defined(NNG_HAVE_SOCKPEERCRED)
+	struct sockpeercred uc;
+		socklen_t    len = sizeof(uc);
```
Comment (gdamore) [inline src/platform/posix/posix_ipcconn.c]: Incorrect indentation.  Please fix.  Thx.

**Comment by gdamore** [comment]
I like it, but can you fix the trivial misindentation noted in the view?

**Comment by opunix** [comment]
@gdamore the indentation is fixed now 👍

**Comment by gdamore** [comment]
The failing test is a known issue, I'm trying to solve for.  So we'll waive that that for this PR.

**Pull request merged**